app/client/src/components/shared/MapWidgets.tsx

```diff
@@ -1052,6 +1066,7 @@ function MapWidgets({
         <div
           id="add-save-data-widget"
           className={addSaveDataWidgetVisible ? '' : 'hidden'}
+          role="region"
```

Comment:
```suggestion
          role="region"
          tabIndex={0}
```

I think we should go ahead and make it so we can navigate to the Add/Save Data Widget with a keyboard.

To do this you will also need to do the following:
1. The suggested change above
2. Add `tabIndex={0}` to the `<Rnd id="add-save-data-widget" ...>` element below.
3. Add the following code to line 1044
```
  useEffect(() => {
    if(!addSaveDataWidgetVisible) return;
    document.getElementById('add-save-data-widget')?.focus();
  }, [addSaveDataWidgetVisible]);
```

Reply:
That's a good call, the other panels don't autofocus but they're only one tab press away, it takes a few more to tab into the Add/Save Data Widget.